lib/fluent/plugin/in_kafka_group.rb

```diff
@@ -176,6 +191,7 @@ def run
             rescue => e
               log.warn "parser error in #{batch.topic}/#{batch.partition}", :error => e.to_s, :value => msg.value, :offset => msg.offset
               log.debug_backtrace
+              reconnect_consumer
```

Comment:
Is this point correct for reconnecting?
This `rescue` is for catching parsing error so kafka brokers should not happen this point.

Reply:
oh good catch .. let me fix and re-test it